.gitignore

```diff
@@ -8,6 +8,7 @@
 
 # production
 /build
```

Comment:
I think we should remove `/build` since it's where webpack-generated stuff goes, based on Create React App configs.

Reply:
Actually I don't think it's that big of a deal, we can change this later, resolving.